Source/Project64-core/N64System/Mips/FlashRam.cpp

```diff
@@ -29,21 +29,21 @@ CFlashram::~CFlashram()
 
 void CFlashram::DmaFromFlashram(uint8_t * dest, int32_t StartOffset, int32_t len)
 {
-    uint8_t FlipBuffer[0x10000];
+    uint8_t FlipBuffer[1 << 16];
```

Comment:
I think you're right; I only changed it from `0x10000` to `1 << 16` just to make GitHub zealously highlight that line to show you where the array was being declared.

It was more to highlight the safety of the change, not really make a better change.  So scratch that.